data/persian/topics/crezq2dg9zwt.json

```diff
@@ -230,7 +230,7 @@
     ],
     "activePage": 1,
     "pageCount": 1,
-    "variantTopicId": null,
+    "": null,
```

Comment:
should this be deleted fully ?

Reply:
Good spot! The double click to highlight didn't work there for some reason 😅